tests/inspect/inspect.go

```diff
@@ -149,6 +152,45 @@ func main() {
 		fmt.Printf("no_new_privs: %v err: %v\n", r1, err)
 	}
 
+	if globalFlags.CheckMknod != "" {
+		/* format: c:5:2:name */
+		dev := strings.SplitN(globalFlags.CheckMknod, ":", 4)
+		if len(dev) != 4 {
```

Comment:
`len(dev) < 4` and `Not enough parameters for mknod` would make the code clearer